### PR TITLE
Moved to mySql DB

### PR DESCRIPTION
**Pull request by vikrud**

### CODE REVIEW

File: modules/user/user.repository.js [hunk 1]
```diff
@@ -58,91 +28,77 @@ class UserRepository {
...
-        const user = await UserModel.findOne({ id: userId });
-
-        if (!user) {
-            throw new CustomError("CANT_FIND_USER_BY_ID");
-        }
-
-        const mongoId = user._id;
-
-        return mongoId;
+        const email = searchParams.email ? "%" + searchParams.email + "%" : "%";
```
Comment (rudenkose) [inline modules/user/user.repository.js]: Use `` for string concatenation

File: modules/user/user.service.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const { userRepository } = require("./user.repository");
 const { encodeJWT } = require("./jwt.config");
+const { CustomError } = require("../../errorHandler");
```
Comment (rudenkose) [inline modules/user/user.service.js:3]: Can't see any using of this

**Review by rudenkose — CHANGES_REQUESTED** [review]
Comments inside

File: modules/user/user.service.js (hunk 2 above)
Comment (vikrud) [reply modules/user/user.service.js:3]: This is used in the UserService.userLogin method when the password does not match

`if (!matchLoginPass) {
            throw new CustomError("PASSWORD_IS_INCORRECT");
        }`

**Review by rudenkose — APPROVED** [review]

**Pull request merged**